index.js

```diff
@@ -21,7 +21,10 @@
 //. import {env, FutureType} from 'fluture-sanctuary-types/index.js';
 //. import {resolve} from 'fluture/index.js';
 //.
-//. const S = sanctuary.create ({checkTypes: true, env: S.env.concat (env)});
+//. const S = sanctuary.create ({
+//.   checkTypes: true,
+//.   env: sanctuary.env.concat (env)
```

Comment:
Trailing comma?

Reply:
I had it there before, https://github.com/fluture-js/fluture-sanctuary-types/compare/18e393167a21c2a66a4a4699e6f9b64ed803882d..2b45ef56ffa6a54d96acbbfa4125bf0f89ff0e6f, but my linter doesn't like it.

Reply:
I should update the linting rules in https://github.com/fluture-js/fluture-template some day.

Reply:
Ah, I think it uses the wrong linting rules. The source is es3 compatible, but the docs don't have to be.